src/env_state.F90

```diff
@@ -643,6 +643,75 @@ subroutine env_state_input_netcdf(env_state, ncid)
 
   end subroutine env_state_input_netcdf
 
+
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+  !> compute saturated vapor pressure (units : Pa) with respective to water
+  !> Formula (10) from [Murphy & Koop, 2004]  (https://doi.org/10.1256/qj.04.94)
+  real(kind=dp) function env_state_saturated_vapor_pressure_water(T)
+
+    !> Temperature (K)
+    real(kind=dp), intent(in) :: T
+
+    real(kind=dp) :: tmp
+
+
+    call warn_assert_msg(571128376, (T > 123) .and. (T < 332), &
+            "The environment temperature is less then 123K or larger than "&
+            "332K, the subroutine env_state_saturated_vapor_pressure_water"&
+            " isn't applicable")
+
+    tmp = 54.842763 &
+        - 6763.22 / T &
+        - 4.210 * log(T) &
+        + 0.000367 * T &
+        + tanh( 0.0415 * (T - 218.8)) &
+            * (53.878 - 1331.22 / T - 9.44523 * log(T) + 0.014025 * T)
+    env_state_saturated_vapor_pressure_water = exp(tmp)
+
+  end function env_state_saturated_vapor_pressure_water
+
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+  !> Compute saturated vapor pressure (units : Pa) with respective to ice
+  !> Formula (7) from [Murphy & Koop, 2004]  (https://doi.org/10.1256/qj.04.94)
+  real(kind=dp) function env_state_saturated_vapor_pressure_ice(T)
+
+    !> Temperature (K)
+    real(kind=dp), intent(in) :: T
+
+    real(kind=dp) :: tmp
+
+    call warn_assert_msg(482130832, T > 110, &
+         "The environment temperature is less then 110K, "&
+         "the subroutine env_state_saturated_vapor_pressure_water"&
+         " isn't applicable")
+
+    tmp = 9.550426 &
+        - 5723.265 / T &
+        + 3.53068 * log(T) &
+        - 0.00728332 * T
+    env_state_saturated_vapor_pressure_ice = exp(tmp)
+
+  end function env_state_saturated_vapor_pressure_ice
+
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+  !> compute saturated vapor pressure (units : Pa) with respective to water
+  real(kind=dp) function env_state_saturated_vapor_pressure_water_2(T)
```

Comment:
I can't seem to find where this is used. it is also a near duplicate to env_state_sat_vapor_pressure (just takes a different argument)